src/app/ui/members-quest-approval/members-quest-approval.component.ts

```diff
@@ -12,16 +10,20 @@ import { environment } from 'src/environments/environment';
 import { PlayerQuestService } from '../player-quest/player-quest.service';
 import { QuestApprovalDialogComponent } from './quest-approval-dialog.component';
 import { RejectReasonDialogComponent } from './reject-reason-dialog/reject-reason-dialog.component';
+import { TeamsService } from 'src/app/teams/teams.service';
+import { FormControl } from '@angular/forms';
 
 @Component({
   selector: 'members-quest-approval',
   templateUrl: './members-quest-approval.component.html',
   styleUrls: ['./members-quest-approval.component.scss']
 })
 export class MembersQuestApprovalComponent implements OnInit {
-
   @Input() user: User;
   questsForApproval$: Observable<PlayerQuest[]>;
+  leadMemberships$: Observable<Membership[]>;
+  private seasonId: string;
+  leadMembershipId = new FormControl('', []);
```

Comment:
you can remove the parameters
```suggestion
  leadMembershipId = new FormControl();
```

Reply:
yup, it's cleaned up